graphene_django/static/graphene_django/graphiql.js

```diff
@@ -136,14 +136,15 @@
     var operationRegex = new RegExp(
       // Look for lines that start with an operation keyword, ignoring whitespace.
       "^\\s*(query|mutation|subscription)\\s+" +
-        // The operation keyword should be followed by the operationName in the GraphQL parameters.
-        graphQLParams.operationName +
         // The line should eventually encounter an opening curly brace.
         "[^\\{]*\\{",
       // Enable multiline matching.
       "m",
     );
```

Comment:
```suggestion
    var operationRegex = new RegExp(
      // Look for lines that start with an operation keyword, ignoring whitespace.
      "^\\s*(query|mutation|subscription)\\s*" +
        // The operation keyword should be followed by whitespace and the operationName in the GraphQL parameters (if available).
        (graphQLParams.operationName ? ("\\s+" + graphQLParams.operationName) : '') +
        // The line should eventually encounter an opening curly brace.
        "[^\\{]*\\{",
      // Enable multiline matching.
      "m",
    );
```

Reply:
Thanks @eabruzzese . Made the change.

Reply:
Gonna try it out quick in my project; one moment.